lua/lvim/core/alpha/dashboard.lua

```diff
@@ -102,31 +102,25 @@ function M.get_sections()
       hl = "Number",
     },
   }
-  local buttons = {}
 
-  local status_ok, dashboard = pcall(require, "alpha.themes.dashboard")
-  if status_ok then
-    local function button(sc, txt, keybind, keybind_opts)
-      local b = dashboard.button(sc, txt, keybind, keybind_opts)
-      b.opts.hl_shortcut = "Include"
-      return b
-    end
-    buttons = {
-      val = {
-        button("f", lvim.icons.ui.FindFile .. "  Find File", "<CMD>Telescope find_files<CR>"),
-        button("n", lvim.icons.ui.NewFile .. "  New File", "<CMD>ene!<CR>"),
-        button("p", lvim.icons.ui.Project .. "  Projects ", "<CMD>Telescope projects<CR>"),
-        button("r", lvim.icons.ui.History .. "  Recent files", ":Telescope oldfiles <CR>"),
-        button("t", lvim.icons.ui.FindText .. "  Find Text", "<CMD>Telescope live_grep<CR>"),
-        button(
-          "c",
-          lvim.icons.ui.Gear .. "  Configuration",
-          "<CMD>edit " .. require("lvim.config"):get_user_config_path() .. " <CR>"
-        ),
+  local buttons = {
+    opts = {
+      hl_shortcut = "Include",
+      spacing = 1,
+    },
+    entries = {
+      { "f", lvim.icons.ui.FindFile .. "  Find File", "<CMD>Telescope find_files<CR>" },
```

Comment:
not sure if using `default_button_opts` is obvious enough, we can also populate the default entries
```suggestion
      {
        "f",
        lvim.icons.ui.FindFile .. "  Find File",
        "<CMD>Telescope find_files<CR>",
        {
          hl_shortcut = "Error",
        },
      },
```

Reply:
I think it's alright